### PR TITLE
OKTA-668502 Create archive area and add PHP content

### PR DESCRIPTION
**Pull request by danielmaharry-okta**
## Description:
**What's changed?** 
- New archive area created.
- All PHP-related content moved to the archive. If standalone article, remove it from navbar
- All PHP-related content removed from conductor.yaml
- Link to archive added to 404 error page

**Is this PR related to a Monolith release?** No

Netlify Preview (Feb 08): https://65c4bce4c322750bf1262ae8--reverent-murdock-829d24.netlify.app/archive/

### Resolves:

* [OKTA-668502](https://oktainc.atlassian.net/browse/OKTA-668502)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/.vuepress/public/img/archive/readme.md](https://github.com/okta/okta-developer-docs/blob/73f793d8e3a09d9d64945ea7c84c8ccfa479af18/packages%2F%40okta%2Fvuepress-site%2F.vuepress%2Fpublic%2Fimg%2Farchive%2Freadme.md) | [88](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a0596c11-e256-48b0-8680-9d9011773cb6) | 100  | 18  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/index.md](https://github.com/okta/okta-developer-docs/blob/73f793d8e3a09d9d64945ea7c84c8ccfa479af18/packages%2F%40okta%2Fvuepress-site%2Farchive%2Findex.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c9f57175-738e-49f2-87bb-687505059729) | 47  | 100  | 47  | 64  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/php/get-started-with-php/index.md](https://github.com/okta/okta-developer-docs/blob/73f793d8e3a09d9d64945ea7c84c8ccfa479af18/packages%2F%40okta%2Fvuepress-site%2Farchive%2Fphp%2Fget-started-with-php%2Findex.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a9ddbc19-fcd4-4d3a-aba8-433b2f0620ed) | 71  | 71  | 45  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/php/protect-your-api/index.md](https://github.com/okta/okta-developer-docs/blob/73f793d8e3a09d9d64945ea7c84c8ccfa479af18/packages%2F%40okta%2Fvuepress-site%2Farchive%2Fphp%2Fprotect-your-api%2Findex.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7e3a6886-47be-4c36-9bfa-524c3579b89e) | 89  | 61  | 59  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/php/sampleapp-oie-redirectauth/index.md](https://github.com/okta/okta-developer-docs/blob/73f793d8e3a09d9d64945ea7c84c8ccfa479af18/packages%2F%40okta%2Fvuepress-site%2Farchive%2Fphp%2Fsampleapp-oie-redirectauth%2Findex.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/02829855-d37c-448c-a3ab-1f86d71cffec) | 62  | 67  | 89  | 99  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/php/sign-into-web-app-redirect/index.md](https://github.com/okta/okta-developer-docs/blob/73f793d8e3a09d9d64945ea7c84c8ccfa479af18/packages%2F%40okta%2Fvuepress-site%2Farchive%2Fphp%2Fsign-into-web-app-redirect%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1f7022fe-a60c-47c7-9eb1-76a0fee1b6be) | 82  | 88  | 71  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/php/simplesamlphp/index.md](https://github.com/okta/okta-developer-docs/blob/73f793d8e3a09d9d64945ea7c84c8ccfa479af18/packages%2F%40okta%2Fvuepress-site%2Farchive%2Fphp%2Fsimplesamlphp%2Findex.md) | [66](https://okta.acrolinx.cloud/api/v1/checking/scorecards/85c146d2-4ef6-401a-97c8-772c214fb5e2) | 54  | 58  | 50  | 64  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/php/validate-access-tokens/index.md](https://github.com/okta/okta-developer-docs/blob/73f793d8e3a09d9d64945ea7c84c8ccfa479af18/packages%2F%40okta%2Fvuepress-site%2Farchive%2Fphp%2Fvalidate-access-tokens%2Findex.md) | [64](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2899b909-024b-4999-bd03-2fa397fae537) | 72  | 72  | 44  | 82  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/code/index.md](https://github.com/okta/okta-developer-docs/blob/73f793d8e3a09d9d64945ea7c84c8ccfa479af18/packages%2F%40okta%2Fvuepress-site%2Fcode%2Findex.md) | [53](https://okta.acrolinx.cloud/api/v1/checking/scorecards/57c4b795-93ca-42c8-9f77-21e08fa78a6c) | 28  | 61  | 36  | 61  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/saml2/prep.md](https://github.com/okta/okta-developer-docs/blob/73f793d8e3a09d9d64945ea7c84c8ccfa479af18/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-sso-integration%2Fmain%2Fsaml2%2Fprep.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b2318365-fadd-4e19-8b01-0ea23d0196ad) | 79  | 66  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcode/examples.md](https://github.com/okta/okta-developer-docs/blob/73f793d8e3a09d9d64945ea7c84c8ccfa479af18/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-grant-type%2Fmain%2Fauthcode%2Fexamples.md) | [68](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6092a53b-fe91-4d67-a575-0df0d91ff603) | 15  | 51  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/quickstart/main/index.md](https://github.com/okta/okta-developer-docs/blob/73f793d8e3a09d9d64945ea7c84c8ccfa479af18/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fquickstart%2Fmain%2Findex.md) | [72](https://okta.acrolinx.cloud/api/v1/checking/scorecards/fe985a17-e554-443c-8abb-993b086751e0) | 67  | 67  | 100  | 67  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/saml-tracer/main/index.md](https://github.com/okta/okta-developer-docs/blob/73f793d8e3a09d9d64945ea7c84c8ccfa479af18/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsaml-tracer%2Fmain%2Findex.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/23ed5dcb-f93f-490d-8c5a-8f60cac0575e) | 50  | 40  | 86  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-id-tokens/main/index.md](https://github.com/okta/okta-developer-docs/blob/73f793d8e3a09d9d64945ea7c84c8ccfa479af18/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-id-tokens%2Fmain%2Findex.md) | [68](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d3c3c3f8-a393-405b-a59d-709150cf8514) | 84  | 64  | 45  | 90  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/errors/404.md](https://github.com/okta/okta-developer-docs/blob/73f793d8e3a09d9d64945ea7c84c8ccfa479af18/packages%2F%40okta%2Fvuepress-site%2Ferrors%2F404.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6c06d382-defa-4f70-98ff-70aea0a1ad51) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/index.md](https://github.com/okta/okta-developer-docs/blob/73f793d8e3a09d9d64945ea7c84c8ccfa479af18/packages%2F%40okta%2Fvuepress-site%2Findex.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/fcd84624-5345-4587-ada9-59559fa8dc3f) | 22  | 22  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.336a0bf2-dff7-4645-8967-9d5299159e2b?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/.vuepress/public/img/archive/readme.md](https://github.com/okta/okta-developer-docs/blob/90701b9b1ef2d563a031e409779428f41cf537ac/packages%2F%40okta%2Fvuepress-site%2F.vuepress%2Fpublic%2Fimg%2Farchive%2Freadme.md) | [88](https://okta.acrolinx.cloud/api/v1/checking/scorecards/959b6305-db55-48a7-a1f6-b7f9267a6d99) | 100  | 18  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/index.md](https://github.com/okta/okta-developer-docs/blob/90701b9b1ef2d563a031e409779428f41cf537ac/packages%2F%40okta%2Fvuepress-site%2Farchive%2Findex.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/df13870c-a9b2-4890-ac25-b74e4d6939a8) | 47  | 100  | 47  | 64  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/php/get-started-with-php/index.md](https://github.com/okta/okta-developer-docs/blob/90701b9b1ef2d563a031e409779428f41cf537ac/packages%2F%40okta%2Fvuepress-site%2Farchive%2Fphp%2Fget-started-with-php%2Findex.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/16e87d42-5b4e-44cf-b03f-b245ad3e81d0) | 71  | 71  | 45  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/php/protect-your-api/index.md](https://github.com/okta/okta-developer-docs/blob/90701b9b1ef2d563a031e409779428f41cf537ac/packages%2F%40okta%2Fvuepress-site%2Farchive%2Fphp%2Fprotect-your-api%2Findex.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b04af04a-1330-447d-a83b-6606630bfa3a) | 89  | 61  | 59  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/php/sampleapp-oie-redirectauth/index.md](https://github.com/okta/okta-developer-docs/blob/90701b9b1ef2d563a031e409779428f41cf537ac/packages%2F%40okta%2Fvuepress-site%2Farchive%2Fphp%2Fsampleapp-oie-redirectauth%2Findex.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/56c3c843-e8df-4e9f-bdc7-f2ecf49a54bd) | 62  | 67  | 89  | 99  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/php/sign-into-web-app-redirect/index.md](https://github.com/okta/okta-developer-docs/blob/90701b9b1ef2d563a031e409779428f41cf537ac/packages%2F%40okta%2Fvuepress-site%2Farchive%2Fphp%2Fsign-into-web-app-redirect%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/72aa6d72-bab2-4617-b6b0-a2a384c2b638) | 82  | 88  | 71  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/php/simplesamlphp/index.md](https://github.com/okta/okta-developer-docs/blob/90701b9b1ef2d563a031e409779428f41cf537ac/packages%2F%40okta%2Fvuepress-site%2Farchive%2Fphp%2Fsimplesamlphp%2Findex.md) | [66](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0634b974-a481-4d83-8d5d-1285433e4f5d) | 54  | 58  | 50  | 64  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/php/validate-access-tokens/index.md](https://github.com/okta/okta-developer-docs/blob/90701b9b1ef2d563a031e409779428f41cf537ac/packages%2F%40okta%2Fvuepress-site%2Farchive%2Fphp%2Fvalidate-access-tokens%2Findex.md) | [64](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d0fc5ef2-ad7a-459b-ab4d-23961767ec98) | 72  | 72  | 44  | 82  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/code/index.md](https://github.com/okta/okta-developer-docs/blob/90701b9b1ef2d563a031e409779428f41cf537ac/packages%2F%40okta%2Fvuepress-site%2Fcode%2Findex.md) | [53](https://okta.acrolinx.cloud/api/v1/checking/scorecards/62654463-f14a-4a3c-8748-178ed22ebbfd) | 28  | 61  | 36  | 61  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/saml2/prep.md](https://github.com/okta/okta-developer-docs/blob/90701b9b1ef2d563a031e409779428f41cf537ac/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-sso-integration%2Fmain%2Fsaml2%2Fprep.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1d669488-358d-44ba-8a0c-4eaf849a3a23) | 79  | 66  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcode/examples.md](https://github.com/okta/okta-developer-docs/blob/90701b9b1ef2d563a031e409779428f41cf537ac/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-grant-type%2Fmain%2Fauthcode%2Fexamples.md) | [68](https://okta.acrolinx.cloud/api/v1/checking/scorecards/20cd59dd-49bd-4f79-a7ac-e09b5c4d8f75) | 15  | 51  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/quickstart/main/index.md](https://github.com/okta/okta-developer-docs/blob/90701b9b1ef2d563a031e409779428f41cf537ac/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fquickstart%2Fmain%2Findex.md) | [72](https://okta.acrolinx.cloud/api/v1/checking/scorecards/453aa428-c9fb-4a58-95f2-a5be99693990) | 67  | 67  | 100  | 67  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/saml-tracer/main/index.md](https://github.com/okta/okta-developer-docs/blob/90701b9b1ef2d563a031e409779428f41cf537ac/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsaml-tracer%2Fmain%2Findex.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7331ebe5-2840-41bc-85bc-6bfd51f3bc69) | 50  | 40  | 86  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-id-tokens/main/index.md](https://github.com/okta/okta-developer-docs/blob/90701b9b1ef2d563a031e409779428f41cf537ac/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-id-tokens%2Fmain%2Findex.md) | [68](https://okta.acrolinx.cloud/api/v1/checking/scorecards/df8c1ef9-76ea-4eb7-9f84-6a3dda5d55a3) | 84  | 64  | 45  | 90  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/errors/404.md](https://github.com/okta/okta-developer-docs/blob/90701b9b1ef2d563a031e409779428f41cf537ac/packages%2F%40okta%2Fvuepress-site%2Ferrors%2F404.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/294b9b06-7863-47f4-9306-062788407cf7) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/index.md](https://github.com/okta/okta-developer-docs/blob/90701b9b1ef2d563a031e409779428f41cf537ac/packages%2F%40okta%2Fvuepress-site%2Findex.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/964480eb-00cd-49bf-8b88-dc8c8d81ae65) | 22  | 22  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.e085b76b-85e2-4f79-acca-2a58e4b10bbf?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/validate-id-tokens/main/index.md [hunk 1]
```diff
@@ -76,13 +76,12 @@ This incurs a network request which is slower to do verification, but can be use
...
-## See also 
+## See also
 
-- A high-level overview of OpenID Connect can be found [here](/docs/concepts/oauth-openid/#openid-connect).
-- The ID tokens are in JSON Web Token (JWT) format, the specification for which can be found here: <https://tools.ietf.org/html/rfc7519>. They are signed using private JSON Web Keys (JWK), the specification for which you can find here: <https://tools.ietf.org/html/rfc7517>.
-- More information about Okta's ID tokens can be found in the [OIDC & OAuth 2.0 API Reference](/docs/reference/api/oidc/#id-token).
\ No newline at end of file
+* A high-level overview of OpenID Connect can be found [here](/docs/concepts/oauth-openid/#openid-connect).
+* The ID tokens are in JSON Web Token (JWT) format, the specification for which can be found here: <https://tools.ietf.org/html/rfc7519>. They are signed using private JSON Web Keys (JWK), the specification for which you can find here: <https://tools.ietf.org/html/rfc7517>.
+* More information about Okta's ID tokens can be found in the [OIDC & OAuth 2.0 API Reference](/docs/reference/api/oidc/#id-token).
```
[the remark this answers is not in the document]
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/docs/guides/validate-id-tokens/main/index.md:87]: fixed

File: packages/@okta/vuepress-site/docs/guides/validate-id-tokens/main/index.md [hunk 2]
```diff
@@ -60,11 +60,11 @@ If the `kid` claim doesn't match, it's possible that the signing keys have chang
 
 You should verify the following:
 
-- The `iss` (issuer) claim matches the identifier of your Okta authorization server.
-- The `aud` (audience) claim should match the Client ID that you used to request the ID Token. This will be the Client ID for the Application you created in Okta.
-- The `iat` (issued at time) claim indicates when this ID token was issued, expressed in Unix time.
-- The `exp` (expiry time) claim is the time at which this token will expire., expressed in Unix time. You should make sure that this time has not already passed.
-- The `nonce` claim value should match whatever was passed when you requested the ID token.
+* The `iss` (issuer) claim matches the identifier of your Okta authorization server.
+* The `aud` (audience) claim should match the Client ID that you used to request the ID Token. This will be the Client ID for the Application you created in Okta.
```
[the remark this answers is not in the document]
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/docs/guides/validate-id-tokens/main/index.md:64]: fixed

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/.vuepress/public/img/archive/readme.md](https://github.com/okta/okta-developer-docs/blob/b59bb028569617d370ccf488583c7420c0e9398b/packages%2F%40okta%2Fvuepress-site%2F.vuepress%2Fpublic%2Fimg%2Farchive%2Freadme.md) | [88](https://okta.acrolinx.cloud/api/v1/checking/scorecards/13660980-c130-4d93-b41d-69f5c3de1bcd) | 100  | 18  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/index.md](https://github.com/okta/okta-developer-docs/blob/b59bb028569617d370ccf488583c7420c0e9398b/packages%2F%40okta%2Fvuepress-site%2Farchive%2Findex.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b967e5ba-0a8e-479e-b368-a72d63ee8de8) | 47  | 100  | 47  | 64  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/php/get-started-with-php/index.md](https://github.com/okta/okta-developer-docs/blob/b59bb028569617d370ccf488583c7420c0e9398b/packages%2F%40okta%2Fvuepress-site%2Farchive%2Fphp%2Fget-started-with-php%2Findex.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0b334c5b-ac38-4f17-8989-b5538126a6bd) | 71  | 71  | 45  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/php/protect-your-api/index.md](https://github.com/okta/okta-developer-docs/blob/b59bb028569617d370ccf488583c7420c0e9398b/packages%2F%40okta%2Fvuepress-site%2Farchive%2Fphp%2Fprotect-your-api%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6abf3aef-6dbc-4ef1-90a1-dfe84c7919a1) | 89  | 62  | 59  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/php/sampleapp-oie-redirectauth/index.md](https://github.com/okta/okta-developer-docs/blob/b59bb028569617d370ccf488583c7420c0e9398b/packages%2F%40okta%2Fvuepress-site%2Farchive%2Fphp%2Fsampleapp-oie-redirectauth%2Findex.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/09f2afb1-8e1b-4fb0-bb70-c9f2cd3044a0) | 62  | 67  | 89  | 99  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/php/sign-into-web-app-redirect/index.md](https://github.com/okta/okta-developer-docs/blob/b59bb028569617d370ccf488583c7420c0e9398b/packages%2F%40okta%2Fvuepress-site%2Farchive%2Fphp%2Fsign-into-web-app-redirect%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/590b381c-7801-4b13-a46c-9566b93cebc2) | 82  | 88  | 72  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/php/simplesamlphp/index.md](https://github.com/okta/okta-developer-docs/blob/b59bb028569617d370ccf488583c7420c0e9398b/packages%2F%40okta%2Fvuepress-site%2Farchive%2Fphp%2Fsimplesamlphp%2Findex.md) | [66](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9fd82f87-3020-4e09-a8a4-c25834ea1fec) | 54  | 58  | 50  | 64  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/archive/php/validate-access-tokens/index.md](https://github.com/okta/okta-developer-docs/blob/b59bb028569617d370ccf488583c7420c0e9398b/packages%2F%40okta%2Fvuepress-site%2Farchive%2Fphp%2Fvalidate-access-tokens%2Findex.md) | [64](https://okta.acrolinx.cloud/api/v1/checking/scorecards/de992056-381c-4fb3-a3e1-7625ece65a03) | 72  | 72  | 44  | 82  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/code/index.md](https://github.com/okta/okta-developer-docs/blob/b59bb028569617d370ccf488583c7420c0e9398b/packages%2F%40okta%2Fvuepress-site%2Fcode%2Findex.md) | [53](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c739d218-7e97-4a1a-9988-c5c9d82251a2) | 28  | 61  | 36  | 61  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/saml2/prep.md](https://github.com/okta/okta-developer-docs/blob/b59bb028569617d370ccf488583c7420c0e9398b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-sso-integration%2Fmain%2Fsaml2%2Fprep.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2730aa7d-a7eb-4500-bd45-df0f693ba258) | 79  | 66  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcode/examples.md](https://github.com/okta/okta-developer-docs/blob/b59bb028569617d370ccf488583c7420c0e9398b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-grant-type%2Fmain%2Fauthcode%2Fexamples.md) | [68](https://okta.acrolinx.cloud/api/v1/checking/scorecards/96c57625-b417-4f88-b960-bda5daa64f6c) | 15  | 51  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/quickstart/main/index.md](https://github.com/okta/okta-developer-docs/blob/b59bb028569617d370ccf488583c7420c0e9398b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fquickstart%2Fmain%2Findex.md) | [72](https://okta.acrolinx.cloud/api/v1/checking/scorecards/41eadc0d-3bbd-4334-a70c-4170848c053e) | 67  | 67  | 100  | 67  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/saml-tracer/main/index.md](https://github.com/okta/okta-developer-docs/blob/b59bb028569617d370ccf488583c7420c0e9398b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsaml-tracer%2Fmain%2Findex.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e73275e2-a091-4fc8-8bb3-f7747e552182) | 50  | 40  | 86  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-id-tokens/main/index.md](https://github.com/okta/okta-developer-docs/blob/b59bb028569617d370ccf488583c7420c0e9398b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-id-tokens%2Fmain%2Findex.md) | [69](https://okta.acrolinx.cloud/api/v1/checking/scorecards/28b6433b-ed00-4c06-9555-ce540e37b797) | 91  | 64  | 45  | 90  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/errors/404.md](https://github.com/okta/okta-developer-docs/blob/b59bb028569617d370ccf488583c7420c0e9398b/packages%2F%40okta%2Fvuepress-site%2Ferrors%2F404.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/959d7d8b-939f-4246-a7c2-369dec81c2d7) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/index.md](https://github.com/okta/okta-developer-docs/blob/b59bb028569617d370ccf488583c7420c0e9398b/packages%2F%40okta%2Fvuepress-site%2Findex.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c5eaf9f2-e22b-4076-8b82-db4c8d6dff5e) | 22  | 22  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.f648361f-0329-4053-b641-d3f669da3f53?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**